backend/btrixcloud/crawls.py

```diff
@@ -318,6 +320,9 @@ async def _resolve_crawl_refs(
             if not crawl.name:
                 crawl.name = config.name
 
+            if not crawl.description:
```

Comment:
Would this ever not be empty? The crawl itself doesn't have a description.. (Possibly also applies to the name?)

Reply:
Good point! Description should always be empty. Name might not be for API list endpoints because it's added in the aggregation, so will keep name as-is.